integration-tests/tests/src/tests/results.ts

```diff
@@ -402,7 +402,7 @@ describe("Results", () => {
       static schema: Realm.ObjectSchema = {
         name: "PersonObject",
         properties: {
-          name: "string",
+          name: { type: "string", mapTo: "NAME" },
```

Comment:
while it's cool that this is specifically also showcasing case sensitvity, could it be useful for us for this to test that this also [also] maps to an entirely different property name?